core/src/main/java/io/ddf/etl/TransformationHandler.java

```diff
@@ -222,14 +218,131 @@ public static String RToSqlUdf(List<String> RExps, List<String> selectedColumns,
 
     String selectStr = "";
     for (String udf : udfs) {
-      String exp = newColToDef.containsKey(udf) ? String.format("%s as %s", newColToDef.get(udf), udf) : String.format(
-          "%s", udf);
+      String exp = newColToDef.containsKey(udf) ?
+          String.format("%s as %s", newColToDef.get(udf), udf) :
+          String.format("%s", udf);
       selectStr += (exp + ",");
     }
 
     return selectStr.substring(0, selectStr.length() - 1);
   }
 
+  /**
+   * Create new columns or overwrite existing ones
+   *
+   * @param newColumnNames       Array of new column names.
```

Comment:
can we limit the number of column and expression to be one at a time, I don't see many use case where DS need to transform multiple columns at a time.